pkg/sources/sources.go

```diff
@@ -240,7 +240,7 @@ type Progress struct {
 // Validator is an interface for validating a source. Sources can optionally implement this interface to validate
 // their configuration.
 type Validator interface {
-	Validate() []error
+	Validate(context.Context) []error
```

Comment:
Why did you add a context here if your new implementation doesn't use it? (And why doesn't this affect other implementations?)

Reply:
good catch and apologies - the commit that included the use of ctx didn't get pushed up - but the Github source does need the context to attempt a call on the ghClient to validate that the auth details from the config file worked.

as far as i can tell, syslog is the only other source to use this Validator interface so just updated that.

Reply:
btw here's the associated change on thog https://github.com/trufflesecurity/thog/pull/1385/files

Reply:
trufflesecurity/thog#1626